frontends/web/src/routes/account/info/buyReceiveCTA.tsx

```diff
@@ -54,14 +54,16 @@ export const BuyReceiveCTA = ({
   const isMobile = useMediaQuery('(max-width: 768px)');
 
   const onExchangeCTA = () => navigate(code ? `/exchange/info/${code}` : '/exchange/info');
-  const onWalletConnect = () => navigate(`/account/${code}/wallet-connect/dashboard`);
+  const onWalletConnect = () => code && navigate(`/account/${code}/wallet-connect/dashboard`);
   const onReceiveCTA = () => {
     if (balanceList) {
       if (balanceList.length > 1) {
         navigate('/accounts/select-receive');
         return;
       }
-      navigate(`/account/${code}/receive`);
+      if (code) {
+        navigate(`/account/${code}/receive`);
+      }
```

Comment:
This looks a bit weird and could be cleaned up by refatoring AddBuyReceiveOnEmptyBalances and BuyReceiveCTA component.

I.e. balanceList is probably not really needed in BuyReceiveCTA and something like a boolean `prop.isOnAccount` or similar may be enough